FilecoinFoundationWeb/filecoin-foundation
# [UXIT-1307] Digest Page Scaffold

## 📝 Description

This PR includes a new `digest` and `digest/[slug]` pages. 

- **Type:** New feature

## 🛠️ Key Changes

- `digest` and `digest/[slug]` pages and related tests and config files
- new cms config schema 
- updated navigation 

## 📌 To-Do Before Merging

- [ ] Remove dummy digest article
- [ ] Maybe also delete `app/_data/cmsConfigSchema.json`?

## 🧪 How to Test
- **Steps to Test:**
  1. Navigate to the content manager and create a new digest article.
  2. Verify that the new digest article appears correctly on the main digest page.
  3. Click on the digest article to open its dedicated digest/[slug] page.
  4. Confirm that the layout, styling, and content display as expected, matching the Figma designs.

## 📸 Screenshots

![screencapture-localhost-3000-digest-2024-08-27-10_30_09](https://github.com/user-attachments/assets/b1909343-1997-4274-ac7c-5cf90a82b382)
![screencapture-localhost-3000-digest-letter-from-the-guest-editor-2024-08-27-10_30_56](https://github.com/user-attachments/assets/0ef4fab3-a8b3-42ff-a5a4-17a849dbdc54)
